### ISSUE TITLE
Design ‘Additional Info’ form variant 

### ISSUE THREAD

**Issue by edrushka** [mozilla/donate.mozilla.org#550]
We need a variation of the form for users arriving at it without the context of snippet, social or email copy. A prime example is Tiles, where we don’t have the space to prime the user with copy before they land on the form.

@lovegushwa has already wireframed this here: https://redpen.io/fva93a5fe9f6116da8

This ticket is to finish the design.

cc @valianttry @ScottDowne @adamlofting

**Comment by valianttry** [comment]
@lovegushwa i left some comments on the Redpen.

**Comment by edrushka** [comment]
Changing this milestone to Nov 6 although this should be done before @lovegushwa gets caught up in London activity.

**Comment by edrushka** [comment]
Copy being written in #633

**Comment by edrushka** [comment]
@ScottDowne or @adamlofting  can you let me know when you're hoping to run this test so I can give @lovegushwa an updated deadline? I feel like we can try the Directory Tile *without* the 'additional info' variant for a while to get a baseline. I believe Directory Tiles go live on Nov 15. Will a week or two give us a good baseline?

**Comment by ScottDowne** [comment]
I'm not super sure what the deadline should be... sooner is better? When we have tile traffic?

I wonder if it could run separately from tests coming from the snippet, allowing us to run multiple non conflicting tests at the same time?

It's a trivial amount of dev work. An hour, maybe less, and I'm using the Scotty Principle here.

@adamlofting Are we going to run this test 50/50 on tile traffic? So half the people coming from the tile get the test and the other half get the default form?

**Comment by adamlofting** [comment]
As soon as (sensbibly and humanely and no need to panickly) possible :+1: 

There isn't a go/no-go deadline for this piece of work as it's an enhancement and would add value even if we shipped it after the campaign launchs. But if it works well it will add cumulative value over the course of the campaign, so the sooner the better really. 

@ScottDowne 
> I wonder if it could run separately from tests coming from the snippet, allowing us to run multiple non conflicting tests at the same time?

Yes, we can/will target A/B tests to this audience specifically, and this can happen at the same time as other tests.

@edrushka 
> Will a week or two give us a good baseline?

Because of the way the A/B test will run, we don't need a baseline - we'll have our control built into the test.

**Comment by edrushka** [comment]
Great. I'll put this at the top of @lovegushwa's queue. Design should be done relatively soon since he's already done a wireframe: https://redpen.io/fva93a5fe9f6116da8

**Comment by lovegushwa** [comment]
@edrushka @ScottDowne @valianttry @adamlofting 

V1 is the baseline, and I think the most direct approach. I'm going to create another version or two as well: 
https://redpen.io/p/kqf242b8a048db367e

**Comment by lovegushwa** [comment]
@edrushka @ScottDowne @valianttry @adamlofting 
Versions 1 - 5 of the ‘Additional Info’ form variant now available for discussion! 
https://redpen.io/p/kqf242b8a048db367e

Let's decide what to move ahead with, or if we should make additional options.

**Comment by edrushka** [comment]
Just a note on feedback. Please use the redpen for clarifying questions or *specific design feedback*. If you'd like to open something up for discussion, let's do that in one of our standups. Redpen isn't a great place for open-ended thoughts and musings as it's too difficult to translate the comment strings into specific design changes. @valianttry and @adamlofting - please indicate in this bug which of the proposed designs Scott should build.

**Comment by adamlofting** [comment]
I'd vote for buildin `V3` first of all, and if we had time later in the campaign to test `V4 branded funnel` (dapper fox!) because it looks amazing.

**Comment by lovegushwa** [comment]
@edrushka @valianttry are we good to convert this to a build ticket for V3 and V4?
@ScottDowne when do you think you will have time to build V3?

**Comment by edrushka** [comment]
@valianttry can you please confirm you're good for us to build V3 and then V4?

**Comment by adamlofting** [comment]
Just adding some data to this bug to share in the call notes for today.

Comparing the conversion rate for snippet and directly tiles (row 1 and 2) below is what we hope this design variant will help with. 

<img width="1017" alt="screenshot 2015-11-23 11 34 44" src="https://cloud.githubusercontent.com/assets/1901816/11335717/a8fed722-91d6-11e5-99f4-874664d9ea18.png">

I encourage us to get V3 live before we start work on V4.

**Comment by lovegushwa** [comment]
@ScottDowne this should give you what you need to build v3
( @edrushka is this the right place this or do we need a build ticket assigned to @ScottDowne ? )

Download the heart SVG:
https://drive.google.com/a/mozilla.com/file/d/0B2qu3ExKWIlvN0VydkR2RF9FZDg/view?usp=sharing

<img width="993" alt="screen shot 2015-11-24 at 3 18 06 pm" src="https://cloud.githubusercontent.com/assets/12228435/11383775/56134522-92bf-11e5-8627-7617f389676e.png">

<img width="994" alt="screen shot 2015-11-24 at 3 18 29 pm" src="https://cloud.githubusercontent.com/assets/12228435/11383783/5e90fdac-92bf-11e5-9287-6f234561438c.png">

**Comment by valianttry** [comment]
@lovegushwa YES v 3 and 4 are the Ones! We've tested an image before and it didn't work very well.... 

Please note the copy in the example is not approved copy. There are variants written and approved by MoCo here:
https://docs.google.com/document/d/1NXhuYivnZH4pWnNz49DieRmIiwQpwXTaO0DPqHEbFIs/edit#heading=h.tixwdvnqq127

**Comment by lovegushwa** [comment]
@edrushka can you turn this into a build ticket for @ScottDowne ?

**Comment by edrushka** [comment]
Build ticket is #941. Thanks for your work on this Jordan!
